packaging/PKGBUILD

```
@@ -0,0 +1,43 @@
+# Maintainer: Kenee Patel  <patelkenee2804 + aur at gmail <.> com>
+pkgname=piston-cli-git # The main package name as showed in the AUR
+_pkgname=piston-cli # this is just a variable used to get the git name
+pkgver=r71.094d30a # the version that will keep syncing because of the pkgver() function below and stay updated with that on git
+pkgrel=1
+pkgdesc="A universal shell supporting 36+ languages, in terminal code editor with syntax highlight, code compiler (36+ languages). It can run code from pastebin links and files too. " # The description of this package that will be shown in AUR.
+arch=('any') # architecture, as this is python based, this can be any.
+url="https://github.com/Shivansh-007/piston-cli.git" # the url to the source on github
+license=('MIT')
+# Depenencies should be mentioned and updated here.
+depends=(	'python-pygments'
+			'python-requests'
+			'python-prompt_toolkit'
+			'python-rich'
+			'python-argparse'
+)
```

Indentation issues ^
```suggestion
depends=(
    "python-pygments"
    "python-requests"
    "python-prompt_toolkit"
    "python-rich"
    "python-argparse"
)

```